cogs/onmyoji.py

```diff
@@ -285,6 +286,153 @@ def shiki_bounty_embed(self, shiki):
     async def tengu(self, ctx):
         """hurrhurrhurr"""
         await ctx.send(file=discord.File('./images/tengu.jpg'))
+
+#=====================Shard Trading===========================
```

Comment:
so personally i dont like this type of comment imo you should structure the code to tell you this rather than doing it here but... w/e

Reply:
its so i can quickly navigate to it from a massive zoomed out view, i remove those when things are done